lighthouse-core/audits/performance-budget.js

```diff
@@ -20,6 +20,14 @@ const UIStrings = {
     =1 {1 request}
     other {# requests}
    }`,
+  /** Table column header for classifying website resources by type, example: "Scripts", "Third-Party", "Stylesheet". */
+  columnResourceType: 'Resource Type',
+  /** Table column header for the count of requests made per resource type. */
+  columnRequests: 'Requests',
+  /** Table column header for the sizes of all network requests of a specific type. */
+  columnTransferSize: 'Transfer Size',
```

Comment:
I faintly recall having a discussion in a PR about whether these should be generic in `i18n.js` or not, but I guess we ended up doing nothing instead 😆 

Anyhoo, transfer size elsewhere is `columnSize` in `i18n.js` maybe we standardize on that and reuse?

Reply:
Made use of `i18n` columns.  I will defer to @khempenius /@hwikyounglee . to decide if they want to keep "Transfer Size" vs. just "Size".  I kept it just to be no-op here, also `resource-summary` also uses "Transfer Size".

Reply:
I'm in favor of keeping it as "Transfer Size". There's a big difference between "transfer size" and "resource size" and I think this is the only place in the UI where we distinguish which interpretation of "size" this uses.

Reply:
Being specific, i.e. 'Transfer Size', SGTM too

Reply:
All things being equal I definitely prefer `Transfer Size` too, the mixture of resource/transfer and column width restrictions make sense elsewhere but if we can let's :)

Reply:
SGTM will keep as is 😄